src/views/DefineCompany.vue

```diff
@@ -158,9 +177,18 @@ export default class DefineCompany extends Mixins(EntityFilterMixin) {
     this.setDefineCompanyStepValidity(this.businessContactFormValid && this.addressFormValid)
   }
 
+  private onFolioNumberChange (folioNumber: string): void {
+    this.setFolioNumber(folioNumber)
+  }
+
   private get showErrors (): boolean {
     return Boolean(this.$route.query.showErrors)
   }
+
+  private get isPremiumAccount (): boolean {
+    // TODO: update to check session when available in common components
+    return Boolean(sessionStorage.getItem('CURRENT_ACCOUNT') === 'PREMIUM')
```

Comment:
In Filings UI we use a session storage keys enum from sbc-common-components. That might be good to use here.

Reply:
Also it would be nice if there was a constant (or enum) for 'PREMIUM'.

Reply:
Just got an update for sbc-common-components, will clean this up, it isn't just an account type string, it actually gives a block of account information. Will add this to the state so it can be used elsewhere

Reply:
Please also add the dependency to the release ticket that contains this feature. (Create UI Rls 0.4 or 0.5 ???)

Reply:
- Updated to use common component keys. 
- I also removed this specific getter and created a global one. 
- The account information is now in the store to make it accessible for other components.
- Account type enum created

Reply:
> Please also add the dependency to the release ticket that contains this feature. (Create UI Rls 0.4 or 0.5 ???)

Updated ticket with the dependency. I think 0.4 should be fine.

Reply:
So this ticket needs Legal API Rls 2.10, as outlined in /bcgov/entity#3682 ?

Reply:
Yes, the api ticket /bcgov/entity#3766 is in 2.10 now.